osu.Game/Screens/Play/HUD/MultiplayerGameplayLeaderboard.cs

```diff
@@ -89,10 +89,13 @@ private void load(OsuConfigManager config, IAPIProvider api)
             {
                 var users = task.GetResultSafely();
 
-                foreach (var user in users)
+                for (int i = 0; i < users.Length; i++)
                 {
-                    if (user == null)
-                        continue;
+                    var user = users[i] ??= new APIUser
```

Comment:
The `??=` assignment is a bit confusing and also doesn't seem to do much here because `users` is thrown away after anyways but sure.

Reply:
It's supposed to be a `??` oops.